installer/win/WixSharpInstaller/Symphony.cs

```diff
@@ -183,18 +199,53 @@ static public void Main(string[] args)
         // Define our own installation flow, using a mix of custom dialogs (defined in their own files) and built-in dialogs
         project.ManagedUI = new ManagedUI();
         project.ManagedUI.InstallDialogs.Add<Symphony.WelcomeDlg>()
+                                        .Add<Symphony.CloseDlg>()
                                         .Add(Dialogs.InstallDir)
                                         .Add(Dialogs.Progress)
                                         .Add<Symphony.ExitDlg>();
         project.ManagedUI.ModifyDialogs.Add(Dialogs.MaintenanceType)
                                        .Add(Dialogs.Progress)
                                        .Add<Symphony.ExitDlg>();
 
+        project.Load += project_Load;
 
+        project.Platform = Platform.x64;
 
         // Generate an MSI from all settings done above
         Compiler.BuildMsi(project);
     }
+
+    static void project_Load(SetupEventArgs e)
```

Comment:
please use proper casing for the method name.

Reply:
this is how it is done in wixsharp examples, I think it is good to follow that convention.